### DISCUSSION TITLE
How to navigate project panel by using `j` or `k`?

### DISCUSSION THREAD

**Discussion by andrew-ares** [zed-industries/zed#7039] [Feature Requests]
Like press `j` to move down and `k` to move up in project panel. I guess this requires some key-bindings related settings, but I can't figure it out.

**Comment by d1y** [comment]
```jsonc
{
    "context": "ProjectPanel && not_editing",
    "bindings": {
      "j": "menu::SelectNext",
      "k": "menu::SelectPrev",
	}
}
```

**Reply by andrew-ares** [reply]
Works like a charm! Thanks!

**Reply by mrnugget** [reply]
Also take a look at the bindings here: https://github.com/zed-industries/zed/issues/4753#issuecomment-1908632636
